pom.xml

```diff
@@ -116,6 +116,41 @@
                         </configuration>
                     </plugin>
                     <plugin>
+				        <artifactId>maven-surefire-plugin</artifactId>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Tab characters found (this is the first occurrence) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=xml%3AIllegalTabCheck)